api/webhook/post.go

```diff
@@ -82,12 +82,13 @@ var baseErr = "unable to process webhook"
 //
 //nolint:funlen,gocyclo // ignore function length and cyclomatic complexity
 func PostWebhook(c *gin.Context) {
-	logrus.Info("webhook received")
-
 	// capture middleware values
 	m := c.MustGet("metadata").(*internal.Metadata)
+	l := c.MustGet("logger").(*logrus.Entry)
 	ctx := c.Request.Context()
 
+	l.Info("webhook received")
```

Comment:
considering the noise, maybe `Debug` level? or instead we add some fields so its helpful

Reply:
i was going back and forth on this one initially, but i think the request logger is good enough for getting request volume on the endpoint. changed to debug in https://github.com/go-vela/server/pull/1145/commits/2c8afeee31314c8095f1782a3b31f31394464f2d.